Client/Frontend/TabBar/SearchViewController.swift

```diff
@@ -44,18 +71,52 @@ class SearchViewController: UIViewController {
         }
     }
 
-    func updateSearchQuery(text: String) {
-        searchText = text
-        tableView.reloadData()
+    private func requerySuggestClient() {
+        // Note that this can race with any pending query requests, but since we hide this
```

Comment:
In the `SearchSuggestClient` you can capture the Alamofire request it is currently executing and cancel it or set a flag so that it will ignore the results.